examples/stream_safetensors_from_gcs.ipynb

```diff
@@ -0,0 +1,173 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stream Model From GCP GCS\n",
```

Comment:
Instead of GCS, it's better to call it "Cloud Storage". Best to check with documentation folks on what the correct acronym is as well.

Reply:
Good idea. Judging from precedent in documentation, "Cloud Storage" is used heavily.